app/imageswap/imageswap.py

```diff
@@ -274,83 +282,88 @@ def swap_image(container_spec):
 
         app.logger.info('ImageSwap Webhook running in "MAPS" mode')
 
-        swap_maps = build_swap_map(imageswap_maps_file)
+        (exact_maps, swap_maps)= build_swap_map(imageswap_maps_file)
 
         app.logger.debug(f"Swap Maps:\n{swap_maps}")
+        app.logger.debug(f"Exact Maps:\n{exact_maps}")
 
-        # Check if Registry portion includes a ":<port_number>"
```

Comment:
The diff is a little bit confusing. Most of the changes are only indentation because of the additional "else:".